### DISCUSSION TITLE
Imperius in Freezetag

### DISCUSSION THREAD

**Discussion by simse15** [PadWorld-Entertainment/worldofpadman#293] [Polls]
So ive been playing with bots for now till the server opens for testing. but i wanted to see how the gamemodes work for now and have noticed that the Imperius in Freezetag is on small maps waaay to Overpowered, with one good shot you can freeze most of the enemys at once wich is not really good in my opinion. So i would suggtest to remove the Imperius from Freezetag mode at least on the small maps such as Anteroom, Kitchen and Padshop.

**Comment by kai-li-wop** [comment]
I just checked the original XMAS mod where FreezeTag was introduced to WoP, and the iMPERiUS wasn't disabled by default, but I remember something like there was a server option to disable it by server admins. I think it would make sense to disable this weapon in FreezeTag since you have to camp for a while to thaw your teammates, which is part of the game's premise. I will change this ticket into a poll discussion, so people can vote for or against it.

**Comment by kai-li-wop** [comment]
Should we disable iMPERiUS in FreezeTag game mode?

Please vote in #294! I was not able to add a poll afterwards when turning the issue into this discussion. Please continue any discussion there
